src/GuestList/GuestListMainPage.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Container, Table } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { renderRsvpStats, createListOfGuests } from './utils';
+
+const GuestListMainPage = () => (
+  <>
+    <Container fluid="md" className="header">
+      <Container>
+        <h1>Betrothed</h1>
+      </Container>
+      <Container>
+        <h1>Guest List</h1>
+      </Container>
+      <Container>
+        <Link to="/dashboard">
+          <Button>Back to Dashboard</Button>
+        </Link>
```

Comment:
More of a design decision than technical feedback, but having a navbar works as a global back to dashboard button, For my bad for not including it in the prototype but quick fix is just removing it.